plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/dto/AbstractImporterDTO.java

```diff
@@ -0,0 +1,194 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2013, Embraer S.A., Budapest University of Technology and Economics
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0 
+ * which accompanies this distribution, and is available at 
+ * http://www.eclipse.org/legal/epl-v10.html 
+ *
+ * Contributors: 
+ *     Krisztián Gábor Mayer - initial API and implementation 
+ *******************************************************************************/
+
+package hu.bme.mit.massif.simulink.api.dto;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import hu.bme.mit.massif.communication.command.MatlabCommandFactory;
+import hu.bme.mit.massif.communication.datatype.Handle;
+import hu.bme.mit.massif.communication.datatype.MatlabString;
+import hu.bme.mit.massif.simulink.BusSelector;
+import hu.bme.mit.massif.simulink.From;
+import hu.bme.mit.massif.simulink.Goto;
+import hu.bme.mit.massif.simulink.GotoTagVisibility;
+import hu.bme.mit.massif.simulink.InPortBlock;
+import hu.bme.mit.massif.simulink.OutPortBlock;
+import hu.bme.mit.massif.simulink.SimulinkModel;
+import hu.bme.mit.massif.simulink.api.Importer;
+import hu.bme.mit.massif.simulink.api.extension.IBlockImportFilter;
+import hu.bme.mit.massif.simulink.api.extension.IParameterImportFilter;
+import hu.bme.mit.massif.simulink.api.util.ISimulinkAPILogger;
+import hu.bme.mit.massif.simulink.api.util.ImportMode;
+
+public abstract class AbstractImporterDTO {
```

Comment:
Do not use "DTO" in name.
Rename class to `AbstractImportData` and package to `data`.